com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/LaunchRequestHandler.java

```diff
@@ -318,7 +359,9 @@ private static String parseMainClassWithoutModuleName(String mainClass) {
 
         List<String> launchCmds = new ArrayList<>();
         launchCmds.add(System.getProperty("java.home") + slash + "bin" + slash + "java");
-        launchCmds.add(String.format("-agentlib:jdwp=transport=dt_socket,server=%s,suspend=y,address=%s", serverMode ? "y" : "n", address));
+        if (StringUtils.isNotEmpty(address)) {
```

Comment:
`private String[] constructLaunchCommands(LaunchArguments launchArguments, boolean serverMode, String address)` can be improved. Because when `address` is null, the value of `serverMode` will not make any difference.  Will it be better to use something like private `String[] constructLaunchCommands(LaunchArguments launchArguments, String debugArgs)`